ccm_web/client/src/models/feature.ts

```diff
@@ -13,10 +13,17 @@ const mergeSectionProps: FeatureDataProps = {
   ordinality: 1
 }
 
-const gradebookToolsProps: FeatureDataProps = {
-  id: 'GradebookTools',
+const canvasGradebookToolsProps: FeatureDataProps = {
+  id: 'CanvasGradebookTools',
   title: 'Gradebook Tools',
-  description: 'Trim the gradebook from Canvas, or trim the gradebook from a third party to correct format',
+  description: 'Trim the gradebook from Canvas',
+  ordinality: 2
+}
+
+const thirdPartygradebookToolsProps: FeatureDataProps = {
+  id: 'ThirdPartyGradebookTools',
+  title: 'Gradebook Tools',
```

Comment:
The title can be changed  to as per the UX [mockups](https://github.com/tl-its-umich-edu/UX-Work/blob/master/ccm/wireframes/gradebook_feature/gradebook_sc_instructor.pdf). But I want to confirm from with melinda on that

Reply:
instead of `route: '/gradebook-other'` it can be names as `/gradebook-thirdparty`

Reply:
@chrisrrowland can you change the title and route as per the mockup. I will approve once done

Reply:
done